### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by AbdulSalamAbro**
*setup desktop version of porfolio website

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -399,3 +401,140 @@ span {
...
+    margin-left: 15px;
+    font-weight: bold;
+  }
+
+  #projects-card {
+    grid-template-columns: repeat(3, 1fr);
+    column-gap: 20px;
+    padding-left: 20px;
+    padding-right: 20px;
+  }
```
Comment (zieeco) [inline styles.css:446]: - Hey, great job, however, let's add some spaces to the side as seen in the images below to improve your design

*Figma design*
image

*your design*
image

File: styles.css [hunk 2]
```diff
@@ -399,3 +401,140 @@ span {
...
+  #desktop-contactform {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    background: url(assets/contactbg.PNG);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    padding-bottom: 200px;
+    margin-top: 60px;
+  }
```
Comment (zieeco) [inline styles.css:433]: - Hey, let's also improve our design here by making sure the background image is well aligned in this form section as seen in the Figma design.


*Figma design*
image

*Your design*
image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @AbdulSalamAbro,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice adding media query ✔️
- Nice trying to implement the desktop design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions

- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -399,3 +401,181 @@ span {
...
+    margin-left: 15px;
+    font-weight: bold;
+  }
+
+  #projects-card {
+    grid-template-columns: repeat(3, 1fr);
+    column-gap: 30px;
+    padding-left: 50px;
+    padding-right: 50px;
+  }
```
Comment (KanzaTahreem) [inline styles.css:446]: - [ ] Good job for adjust the padding 👍👍 While fixing the padding, I noticed that the column gap is more than it was present in the previous review. Kindly fix this 🤓⛏ You can do this by changing the `column-gap` to `20px`

Your design before | Your design now
---| ---|
image | image

File: styles.css [hunk 4]
```diff
@@ -399,3 +401,181 @@ span {
...
+
+  form textarea {
+    padding: 10px;
+    padding-right: 240px;
+    margin-left: auto;
+    margin-right: auto;
+    color: #42526e;
+    border: 1px solid #60c095;
+    border-radius: 4px;
+  }
```
Comment (KanzaTahreem) [inline styles.css:565]: _Excellent job for improving the contact section. Well done 👏👏_
- [ ] Good job for fixing the background image 🥇🥇 I noticed that two input fields are present for the name. But in the previous review, only one field was present (for name) 📌Also, the `textarea` was almost aligned with the other fields. Kindly make sure to remove the field for tha last name as it was present previously. This is also mentioned in the project requirements 📁
- [ ] Also, kindly align the `textarea` with other fields and adjust it's height as well according to how it was present in the last review ⛏🤓

Your design previously | Your design now
---| ---|
image | image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @AbdulSalamAbro 

Good job so far! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕


N/A

Cheers and Happy coding!👏👨‍💻💻

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @AbdulSalamAbro,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**